test819.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+from qibo.backends import GlobalBackend
+
+from qibolab import Platform
+from qibolab.execution_parameters import (
+    AcquisitionType,
+    AveragingMode,
+    ExecutionParameters,
+)
+from qibolab.instruments.qblox.controller import QbloxController
+from qibolab.platform import NS_TO_SEC
+from qibolab.pulses import PulseSequence
+from qibolab.sweeper import Parameter, Sweeper, SweeperType
+
+GlobalBackend.set_backend("qibolab", "spinq10q")
+platform: Platform = GlobalBackend().platform
+controller: QbloxController = platform.instruments["qblox_controller"]
+
+sequence = PulseSequence()
+ro_pulses = {}
+
+qid = 0
+qubit = platform.qubits[qid]
+qubits = {qid: qubit}
+
+ro_pulse = platform.create_qubit_readout_pulse(qid, start=0)
+ro_pulse.frequency = int(2e9)
```

Comment:
This one I changed explicitly, otherwise I couldn't get an IF, since it was too far from the LO frequency (apparently)